### ISSUE TITLE
PROPFIND /: Cannot parse XML response (207)

### ISSUE THREAD

**Issue by andrewchenshx** [laurent22/joplin#4236]
## Environment
Joplin 1.4.19 (prod, win32)

Client ID: 73aaf672bfdf42eea36dd958aabf0bf6
Sync Version: 2
Profile Version: 34
Keychain Supported: Yes

Revision: de123ee58 (master)

## Steps to reproduce

1. option -> Synchronisation -> check Synchronisation configuration
Success! Synchronisation configuration appears to be correct.
2. back to main window
3. press "Synchronize" in the left-bottom corner

`
Completed: 20/12/2020 00:47
Last error: Error: PROPFIND /: Cannot parse XML response (207): <?xml version="1.0" encoding="utf-8"?> <D:multistatus xmlns:D="DAV:" xmlns:ns1="http://owncloud.org/ns" xmlns:ns0="DAV:"> <D:response xmlns:lp1="DAV:" xmlns:lp2="http://apache.org/dav/props/"> <D:href>/joplin/</D:href> <D:propstat> <D:prop> <lp1:getlastmodified>Sat, 19 Dec 2020 16:30:35 GMT</lp1:getlastmodified> <lp1:resourcetype><D:collection/></lp1:resourcetype> </D:prop> <D:status>HTTP/1.1 200 OK</D:status> </D:propstat> </D:response> <D:res
`

## Describe what you expected to happen

1.completed without error

## Logfile

[-1608396421809.log](https://github.com/laurent22/joplin/files/5719842/-1608396421809.log)

**Comment by laurent22** [comment]
Looks like something is misconfigured if the XML cannot be parsed at all. For example, the file should start with something like `<?xml version="1.0" encoding="utf-8" ?>`

**Comment by andrewchenshx** [comment]
@laurent22 ,
actually, the XML response started with 
`<?xml version="1.0" encoding="utf-8"?>`

i pasted the whole error message, while the above was NOT display(I updated the issue, inserting the message as code, now it displays properly) .
if u don't mind, pls the log for comfirmation.

**Comment by lenov8** [comment]
Joplin 2.1.8 (prod, win32)

Client ID: 07a5cafc645e410c99036382c1fe594c
Sync Version: 2
Profile Version: 39
Keychain Supported: Yes

Revision: 94ea177b7
Exact same issue with same error message as reported in this thread. Issue occurs on Windows 10 computers as well as Linux Ubuntu vm's. Tried the current version's advanced sync option to, "Re-upload local data to sync target" but after uploading for two days, when it synchronized I got the error message again.
